### PR TITLE
Switch to yt-dlp

### PR DESCRIPTION
**Pull request by xibr**
Replace youtube-dl with yt-dlp #117

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,3 +131,4 @@ try {
 ## Docs
  *  Though not required for just using this library, documentation on building python for android can be seen [here](BUILD_PYTHON.md). Same for ffmpeg [here](BUILD_FFMPEG.md). Alternatively, you can use pre-built packages from [here (android5+)](http://termux.net/dists/stable/) or [here (android7+)](https://bintray.com/termux/termux-packages-24).
  * youtubedl-android uses lazy extractors based build of youtube-dl ([youtubedl-lazy](https://github.com/yausername/youtubedl-lazy/))
```
Comment (yausername) [inline README.md:133]: might be better to just remove this line to avoid confusion for newcomers to the repo.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # youtubedl-android
-Android library wrapper for [youtube-dl](https://github.com/rg3/youtube-dl) executable
+Android library wrapper for [yt-dlp](https://github.com/yt-dlp/yt-dlp) executable
```
Comment (yausername) [inline README.md:2]:
```suggestion
Android library wrapper for [yt-dlp](https://github.com/yt-dlp/yt-dlp) (formerly [youtube-dl](https://github.com/rg3/youtube-dl)) executable
```

File: README.md (hunk 1 above)
Comment (yausername) [reply README.md:133]: or something like this
```suggestion
 * youtubedl-android uses lazy extractors based build of yt-dlp - [ytdlp-lazy](https://github.com/xibr/ytdlp-lazy) (formerly [youtubedl-lazy](https://github.com/yausername/youtubedl-lazy/))
```

**Review by yausername — APPROVED** [review]
LGTM. left a couple of minor comments.
Feel free to merge.
Thanks.

**Comment by sebma** [comment]
Should solve issue [#27](https://github.com/yausername/dvd/issues/27)

**Comment by xibr** [comment]
Yes, I am working on it now.

**Comment by sebma** [comment]
@xibr Yes. Thanks for your work by the way :+1:

**Pull request merged**